### DISCUSSION TITLE
Extension not working for me

### DISCUSSION THREAD

**Discussion by ridhika1628** [joachimdalen/azdevops-auto-state#77] [Q&A]
Hi @joachimdalen - Have installed this extension in my organization and used a preset rule i.e. **update the PBI state to committed when task gets in progress**. However it's not working when am trying to use it in **existing PBI.**
Steps I followed -:

Add and enable preset Rule in my project
Create new child task under PBI which is in New state.
Update the state of Child task from 'To Do' to 'In progress'
Nothing happened in my parent PBI

Note - I am trying to achieve this under backlogs in azure devops.

Your response will be highly appreciated.

<img width="779" alt="Capture" src="https://github.com/joachimdalen/azdevops-auto-state/assets/65011651/6c94fb8b-dcb4-498e-ae07-5406230095b6">

<img width="160" alt="azurebacklog" src="https://github.com/joachimdalen/azdevops-auto-state/assets/65011651/ffe45e30-3abe-46fc-876d-d5ffdd747cb2">

**Comment by joachimdalen** [comment]
Hi,

I'll see if I get some time to look further into this today or tomorrow. I'm not sure right now why it's not working for you.

**Comment by joachimdalen** [comment]
I tested with the following config in a new project and works as it should.  Could you try to remove `Children Lookup` from the `In Progress -> Commited` rule?

![Screenshot from 2023-09-18 19-01-18](https://github.com/joachimdalen/azdevops-auto-state/assets/15696189/4eb7ace8-4c44-4171-ab19-d71ac0c7f7fe)


Please note: The parent state will not show as updated in the backlog view until you do a full refresh of the page or open the parent work item.

**Reply by ridhika1628** [reply]
Hey @joachimdalen 
Thank you so much for looking into this. I get the issue now actually we have applied rules on product backlog item due to which it is clashing.

Once I disabled those rules it started working. However we need to have those rules as well. Can you please suggest if I can use your extension as well as attached rules at the same time?

Your help would be greatly appreciated!

<img width="754" alt="PBI Committed state rules" src="https://github.com/joachimdalen/azdevops-auto-state/assets/65011651/4431947c-be4c-4ec9-bec6-9fdf61444153">

**Reply by joachimdalen** [reply]
The only field that AutoState updates is the `State` field, so with these rules it would probably cause issues if they block saving the work item. For AutoState to work, it looks like those rules would need to be disabled.

**Comment by ridhika1628** [comment]
Hi @joachimdalen - Found 1 bug in this as whenever we click on only save the task, it won't update the Parent item state. However as soon as we click on save and close the child task and make full refresh of page, it works. Can you please apply fix for this.
